wire/carrier.go

```diff
@@ -0,0 +1,38 @@
+package wire
+
+// ProtobufCarrier is a DelegatingCarrier that uses protocol buffers as the
```

Comment:
can't find the comment now, but DelegatingCarrier is already being used: https://github.com/cockroachdb/cockroach/blob/master/util/tracing/span.go